digits/job.py

```diff
@@ -266,3 +266,10 @@ def is_read_only(self):
         Returns False if this job can be edited
         """
         return False
+
+    def parent_jobs(self):
+        """
+        Returns the list of the jobs on which the job
+        depends
+        """
+        raise NotImplementedError('Implement me!') #@TODO
```

Comment:
you could return an empty list to avoid having to catch the exception in the caller?